src/structures/Guild.js

```diff
@@ -534,6 +534,30 @@ class Guild extends AnonymousGuild {
     );
   }
 
+  /**
+   * The maximum bitrate available for this guild
+   * @type {number}
+   * @readonly
+   */
+  get maximumBitrate() {
+    if (this.features.includes('VIP_REGIONS')) {
+      return 384000;
+    } else {
+      switch (this.premiumTier) {
+        case PremiumTiers.indexOf(this.premiumTier) === 0:
+          return 96000;
+        case PremiumTiers.indexOf(this.premiumTier) === 1:
+          return 128000;
+        case PremiumTiers.indexOf(this.premiumTier) === 2:
+          return 256000;
+        case PremiumTiers.indexOf(this.premiumTier) === 3:
+          return 384000;
+        default:
+          return 96000;
+      }
+    }
```

Comment:
```suggestion
    if (this.features.includes('VIP_REGIONS')) {
      return 384000;
    }

    switch (this.premiumTier) {
      case PremiumTiers.NONE:
        return 96000;
      case PremiumTiers.TIER_1:
        return 128000;
      case PremiumTiers.TIER_2:
        return 256000;
      case PremiumTiers.TIER_3:
        return 384000;
      default:
        return 96000;
    }
```

Alternatively, we can also define an object somewhere and do the following:

```javascript
return this.features.includes('VIP_REGIONS') ? 384_000 : PremiumBitrates[this.premiumTier];
```